addons/website/views/snippets/s_banner.xml

```diff
@@ -6,23 +6,20 @@
         <div class="container">
             <div class="row o_grid_mode" data-row-count="11">
                 <div class="justify-content-center o_grid_item g-height-8 g-col-lg-5 col-lg-5" data-name="Box" style="z-index: 1; grid-area: 2 / 1 / 10 / 6;">
-                    <h1 class="display-3">Sell Online. <strong>Easily.</strong></h1>
-                    <p class="lead">This is a simple hero unit, a simple jumbotron-style component for calling extra attention to featured content or information.</p>
-                    <div class="s_hr pb32 pt4" data-snippet="s_hr" data-name="Separator">
-                        <hr class="w-100 mx-auto"/>
-                    </div>
+                    <h1 class="display-3">Unleash your <strong>growth</strong> potential.</h1>
```

Comment:
As discussed on discord ;)
```suggestion
                    <h1 class="display-3">Unleash your <strong>potential</strong>.</h1>
```
| this PR (`lg` screen)| proposal (`lg` screen) |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/b4e26a30-a648-4c65-8c97-dac6406587cb) | ![image](https://github.com/user-attachments/assets/4a985ff5-0e42-47e8-af49-9aa70bfbc766) |